scripts/dev-docs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ev # exit when error
+
+npm run dev &
+cd docs && bundle install && bundle exec guard
```

Comment:
You could move the `bundle install` to an npm `postinstall` script. There's no real need to run it on every `guard` launch.